css/src/core/bare-elements.scss

```diff
@@ -32,3 +32,15 @@ fieldset {
 	min-width: 0; // as a replaced element min-width:0 is required to enable it to work in responsive layouts
 	border: none;
 }
+
+::target-text {
+	background-color: $code-highlight-background;
+}
+
+:where(a)::target-text {
```

Comment:
are latter two needed now with the new bg color?

Reply:
Just tried it out and removed those sections